src/main/kotlin/dev/minn/jda/ktx/interactions/components/buttons.kt

```diff
@@ -67,9 +69,12 @@ fun button(
     id: String,
     label: String? = ButtonDefaults.LABEL,
     emoji: Emoji? = ButtonDefaults.EMOJI,
+    uniqueId: Int = -1,
     style: ButtonStyle = ButtonDefaults.STYLE,
     disabled: Boolean = ButtonDefaults.DISABLED,
-) = Button.of(style, id, label, emoji).withDisabled(disabled)
+) = Button.of(style, id, label, emoji)
+    .withDisabled(disabled)
+    .applyIfNotNull(uniqueId, Button::withUniqueId)
```

Comment:
In this situation, `Button::withUniqueId` will always be applied due to the fact that `uniqueId` is type `Int` and is not nullable. Furthermore, the `withUniqueId` method will always throw an `IllegalArgumentException` when the default value of -1 is used.